docker/training/dockerfile.ctr

```diff
@@ -3,180 +3,307 @@ ARG IMAGE=nvcr.io/nvidia/tensorflow:22.02-tf2-py3
 FROM ${IMAGE}
 
 # Args
+ARG CORE_VER=v0.1.1
 ARG NVTAB_VER=main
 ARG MODELS_VER=main
 ARG HUGECTR_VER=master
-ARG HWLOC_VER=2.4.1
-
-# Envs
-ENV CUDA_SHORT_VERSION=11.6
-ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/cuda/lib64:/usr/local/cuda/extras/CUPTI/lib64:/usr/local/lib:/repos/dist/lib
-ENV CUDA_HOME=/usr/local/cuda
-ENV CUDA_PATH=$CUDA_HOME
-ENV CUDA_CUDA_LIBRARY=${CUDA_HOME}/lib64/stubs
-ENV PATH=${CUDA_HOME}/lib64/:${PATH}:${CUDA_HOME}/bin
-ENV PATH=$PATH:/usr/lib/x86_64-linux-gnu/
-RUN ln -s /usr/local/cuda/lib64/stubs/libcuda.so /usr/local/cuda/lib64/stubs/libcuda.so.1
-
-# Install system packages
-ENV DEBIAN_FRONTEND=noninteractive
-RUN apt update -y --fix-missing && \
-    apt install -y --no-install-recommends software-properties-common && \
+
+ARG CMAKE_VER=3.22.3 \
+    HADOOP_VER=3.3.2 \
+    HIREDIS_VER=1.0.2 \
+    HWLOC_VER=2.4.1 \
+    PROTOC_VER=3.19.4 \
+    RDKAFKA_VER=1.8.2 \
+    REDIS_PP_VER=1.3.3 \
+    ROCKSDB_VER=6.29.3
+
+ARG INSTALL_HDFS=true \
+    BUILD_HADOOP=true
```

Comment:
According to the discussion today - I think we should be defaulting these to false ?

Reply:
Correct. Will make them false. That was a remainder of testing. It takes 1+ hours to generate the image. So I didn't reset them.

Will do so and mark this as resolved as soon as it has been done.